src/common/cups_print.h

```diff
@@ -59,15 +59,12 @@ typedef struct dt_print_info_t
   dt_paper_info_t paper;
 } dt_print_info_t;
 
-// is some printers are available
-gboolean is_printer_available(void);
+// Asynchronous printer discovery, cb will be called for each printer found
```

Comment:
No, this is true since now this is done on a dt background job. cupsEnumDests() won't help as it returns only when all dest have been found. there is no support in cups for asynchronous printer discovery.